products/paas/domains.md

```diff
@@ -108,3 +108,69 @@ The procedure of adding CNAME record to DNS database depending on your DNS servi
 * [Namecheap](https://www.namecheap.com/support/knowledgebase/article.aspx/9646/2237/how-to-create-a-cname-record-for-your-domain){:target="_blank"}
 
 If you don't find your DNS provider in the list provided try to get this information on the providers website or by contacting your provider support.
+
+### Troubleshooting
+
+Firs of all you need to check if you have added CNAME to your domain correctly:
+
+Use dig command for this:
```

Comment:
Use [Google Admin Toolbox](https://toolbox.googleapps.com/apps/dig/){:target="_blank"} or "dig" command if your OS system is Linux.